nixos/modules/services/networking/ntp/chrony.nix

```diff
@@ -85,6 +88,33 @@ in
         '';
       };
 
+      enableRTCTrimming = mkOption {
+        type = types.bool;
+        default = true;
+        description = lib.mdDoc ''
+          Enable tracking of the RTC offset to the system clock and automatic trimming.
+          See also [](#opt-services.chrony.autotrimThreshold)
+
+          ::: {.note}
+          This is not compatible with the `rtcsync` directive, which naively syncs the RTC time every 11 minutes.
+
+          Tracking the RTC drift will allow more precise timekeeping,
+          especially on intermittently running devices, where the RTC is very relevant.
+          :::
+        '';
+      };
+
+      autotrimThreshold = mkOption {
+        type = types.ints.positive;
+        default = 30;
```

Comment:
30 seconds of maximum drift if the computer is offline seems like a lot, perhaps 10 is a better default?

Reply:
Keep in mind that chrony will sync the system clock accurately, as it uses the recorded RTC drift/offset.
So at maximum, the system clock will be less than 30 seconds off *until* chrony starts, so a few seconds after boot it'll hopefully be more accurate than if it was just synced on time on every shutdown/every 11 minutes.

Generally letting chrony have more range until it needs to correct the drift will presumably result in a more accurate drift approximation.
As the RTC does not have the ability to fine tune the rate, it relies on this offset.

This is the value shown in the manpage example and I am unsure if it'd be better to set it to something different.